cartoframes/core/cartodataframe.py

```diff
@@ -4,17 +4,39 @@
 
 
 class CartoDataFrame(GeoDataFrame):
-
     def __init__(self, *args, **kwargs):
+        """
+        A CartoDataFrame object is a
+        `geopandas.GeoDataFrame <http://geopandas.org/reference/geopandas.GeoDataFrame.html>`_
+        that has a column with geometry. It extends the GeoDataFrame object to read and
+        write data from CARTO, adding wrappers when necessary.
+        """
```

Comment:
I'm not sure if the constructor `docstring` is being generated, could we double check? In other classes I've added the docs of the constructor just below the class definition.

Reply:
Adding another comment, but it applies to the rest of the methods of the class. Should we document the *args and **kwargs arguments?

I don't know if it's a good practice or not, but in terms of documentation I think we should be the more explicit possible with the arguments. We might even consider to drop *args and **kwargs in favour of the actual arguments.

Reply:
Let's add also for each method these sections:

- Args
- Returns
- Raises (let's try to dig a little bit in the code and expose there the most commons exceptions you can get)

Reply:
I haven't added anything related with the *args and **kwargs, I think we should tackle this in a different task 👍